modules/influxdb/influxdb_test.go

```diff
@@ -48,6 +50,29 @@ func TestV2Container(t *testing.T) {
 	}
 }
 
+func TestWithWaitStrategy(t *testing.T) {
```

Comment:
question: This test passes without issue for me without any modifications, so I'm not sure I'm fully following as it looks you can customisation is already possible?

Reply:
- In my case, even the current default strategy sometimes passes. This seems to depend on the machine's available resources, as it consistently fails on my low-resources Jenkins setup.

- Unfortunately, full customization is currently not possible because WaitingFor is overridden, for example, in [this line](https://github.com/testcontainers/testcontainers-go/blob/e94530939173164b80faf6dd5870410cd3caf7fe/modules/influxdb/influxdb.go#L75)

Reply:
suggestion: That last point seems to be the thing that should be fixed, and if the health endpoint is always available switching to that as the default would make a lot of sense.

In general we're trying to move away from `wait.ForLog` as its fragile.

Reply:
I checked the health check response for InfluxDB 1, and it looks like this:
```json
{
  "checks": [],
  "message": "ready for queries and writes",
  "name": "influxdb",
  "status": "pass",
  "version": "1.8.10"
}
```
Based on this, I've unified the default strategy for both versions of InfluxDB, simplifying it to two default strategies: one when configuration files are provided and one when they are not. I also removed the previously added tests since the health check verification is now covered by the standard tests.